content/admin/onboard_team/CreateUsers.md

```diff
@@ -4,41 +4,108 @@ description: "Creating Users in Contrast TeamServer"
+
+* First name 
+* Last name 
+* Email 
+* Organization 
+* Organization role <br> (**View**, **Edit**, **Rules admin**, or **Admin**)
+
+> **Note:** The **Email** field appears as **Username** for users who have [LDAP authentication](installation-setupauth.html) configured. 
+
+You may also include the following information for each user in the spreadsheet: 
+
+* Email Activation <br> (If "None", default is **Required Password**)
+* System Administration <br> (Default is **None**)
+* Application access group <br> (View, Edit, Rules Admin, Admin or custom group name(s); default is **Unassigned**)
+* Date Format <br> (Default is the organization setting, such as **MM/dd/YYYY**)
+* Time Format <br> (Default is the organization setting, such as **hh:mm a**)
+* Time zone <br> (Default is your time zone)
+* Access <br> (Default is **On**)
+* API only <br> (Default is **Off**)
+
+> **Note:** Hover over the upload icon for a reminder of required file formats and information fields. You can also download a **CSV template** by clicking the link in the tooltip. 
+
+Once the spreadsheet upload is in progress, you can leave the page and continue with other tasks in Contrast. If the upload is successful, Contrast shows you a confirmation message with the number of users uploaded. If the upload failed, Contrast shows you an error message with the source of the error on the spreadsheet.
+
+## Organization Settings
+
+To create users as an Organization Administrator, go to the **User menu > Organization Settings > Users tab**. 
+
+### Individual users
+
+To add a single user, click the button to **Add User** above the grid, and complete the following steps: 
+
+* Enter the user's **First Name**, **Last Name** and **Email Address** in the provided fields. 
+* Choose the user's **Organization Role** in the dropdown menu. 
+* Select an **Application Access Group** to which to add the user in the dropdown menu, if desired. 
+* Choose **Date Format**, **Time Format** and **Time Zone** settings in the dropdown menus. 
+* If you want to disable the user's access to your organization in the Contrast interface, use the **Access** toggle. (The user has access by default.)
+* Check the box if you want the user to have **API Only** access. (The user will have access Contrast's REST API, but won't have access to the Contrast interface.)
+
+<a href="assets/images/Create_User.png" rel="lightbox" title="Add a user as an Organization Administrator"><img class="thumbnail" src="assets/images/Create_User.png"/></a>
+
+### Multiple users
+
+To bulk add users, click the upload icon above the grid to import a spreadsheet with the users' information. The spreadsheet must be CSV format, and include the following fields for each user. **All field headings and values in the spreadsheet must be formatted as shown.** 
+
+* First name 
```

Comment:
"First name" and "Last name" I have as "First Name" and "Last Name" (capital 'N').